README.md

```diff
@@ -1,17 +1,52 @@
-# eZ Dataflow
+# Code-Rhapsodie EzDataflow Bundle
 
-The eZ Dataflow bundle is intent to manage external data sources import into your EZPlatform instance.
+EzDataflowBundle is a bundle to integrate the Code-Rhapsodie DataflowBundle into eZ Platfom 2.0+.
+These dataflows can be piloted from an UI integrated in eZ Platform Admin UI.
+The EzDataflow bundle is intended to manage content imports from external data sources.
+
+## User Interface (UI)
+
+The UI lets you create workflow processes from any defined `DataflowTypes`, and set that process options.
+
+Processes can be set to run either:
+- as soon as possible, meaning the next time the queue is processed
+- at a given date and time
 
 ## Installation
 
 ### Step 1: Install the bundle via composer
 
 ```shell script
-$ composer require code-rhapsodie/ezdataflow
+$ composer require code-rhapsodie/ezdataflow-bundle
 ```
 
 ### Step 2: Enable the bundle
 
+#### Symfony 4 (new tree)
+
+For Symfony 4, add `CodeRhapsodie\DataflowBundle\CodeRhapsodieDataflowBundle::class => ['all' => true],`
```

Comment:
For Symfony 4, add those two lines in your `config/bundles.php` file : 
```
<?php
...
```

Reply:
Can't we have a recipe for this ? 🤔